### PR TITLE
iOS8 deprecation changes

### PR DESCRIPTION
**Pull request by Harinder**
##### Description
- iOS8 deprecation from Janrain library
NOTE: https://github.com/DocuSignDev/engage.iphone/pull/2 has changes for both update to 5.0.4, as well as iOS8 deprecation

##### JIRA issue(s)
https://jira.corp.docusign.com/browse/IOS-13679

##### Reviewers
@thomas-dang , please review this PR

### CODE REVIEW

File: Janrain/JREngage/Classes/JRProvidersController.m [hunk 1]
```diff
@@ -262,55 +262,61 @@ - (void)viewDidAppear:(BOOL)animated
     [super viewDidAppear:animated];
 }
 
-- (void)alertView:(UIAlertView *)alertView clickedButtonAtIndex:(NSInteger)buttonIndex
-{
-    [sessionData triggerAuthenticationDidTimeOutConfiguration];
-}
```
Comment (thomas-dang) [inline Janrain/JREngage/Classes/JRProvidersController.m]: Just curious, why was this method removed?

Comment (thomas-dang) [inline Janrain/JREngage/Classes/JRProvidersController.m]: Ah, nevermind. iOS 8 deprecation related.

File: Janrain/JREngage/Classes/JRUserInterfaceMaestro.m [hunk 2]
```diff
@@ -482,55 +482,67 @@ - (id)initWithNibName:(NSString *)nibNameOrNil bundle:(NSBundle *)nibBundleOrNil
...
-    [myPopoverController presentPopoverFromBarButtonItem:barButtonItem
-                                permittedArrowDirections:direction animated:YES];
+    // [myPopoverController presentPopoverFromBarButtonItem:barButtonItem
+    //                             permittedArrowDirections:direction animated:YES];
+    
+    
+    
+    self.myPopoverController.barButtonItem = barButtonItem;
+    self.myPopoverController.permittedArrowDirections = direction;
+    [self presentViewController:myNavigationController animated:TRUE completion:^{
```
Comment (thomas-dang) [inline Janrain/JREngage/Classes/JRUserInterfaceMaestro.m]: Should we use `YES` here instead of `TRUE`?

**Comment by thomas-dang** [comment]
@Harinder, completed my review. Just one comment.

**Comment by Harinder** [comment]
@thomas-dang, done with changes. Please take an another quick look.

**Comment by thomas-dang** [comment]
Looks good

**Pull request merged**